src/app/account/reset-password/verification-modal.tsx

```diff
@@ -0,0 +1,25 @@
+import { Paper, Text, CloseButton, Group, Stack } from '@mantine/core'
+import Link from 'next/link'
+
+export function VerificationModal({ onCompleteAction, onBack }: { onCompleteAction: () => void; onBack: () => void }) {
```

Comment:
to me this isn't really a `Modal` because it doesn't appear onto of a other content, it's a `Panel`